sdks/python/apache_beam/io/gcp/pubsub.py

```diff
@@ -338,7 +325,7 @@ def display_data(self):
 
   def reader(self):
     raise NotImplementedError(
-        'PubSubPayloadSource is not supported in local execution.')
+        'PubSubSource is not supported in local execution.')
```

Comment:
PubSubSource is an implementation detail. "Reading form Cloud Pub/Sub is not supported in local execution" ?